src/main/java/at/hannibal2/skyhanni/features/fishing/tracker/SeaCreatureTracker.kt

```diff
@@ -151,10 +157,25 @@ object SeaCreatureTracker {
         tracker.renderDisplay(config.position)
     }
 
+    @SubscribeEvent
+    fun onRepoReload(event: RepositoryReloadEvent) {
+        val data = event.getConstant<ItemsJson>("Items")
+        armorNames.clear()
+        armorNames.addAll(data.trophy_armors)
```

Comment:
As per my previous suggestion:
```suggestion
        armorNames = data.trophy_armors.toSet()
```